pulp_smash/tests/pulp3/pulpcore/api_v3/plugin_involved/test_repo_version.py

```diff
@@ -416,8 +429,9 @@ def test_filter_invalid_version(self):
                 {'number__gte': criteria, 'number__lte': criteria},
                 {'number__range': ','.join((criteria, criteria))}):
             with self.subTest(params=params):
-                versions = get_versions(self.repo, params)
-                self.assertEqual(len(versions), 0, versions)
+                with self.assertRaises(HTTPError):
+                    versions = get_versions(self.repo, params)
+                    self.assertEqual(len(versions), 0, versions)
```

Comment:
An HTTP 400 is returned when calling `get_versions()`, right? If so, this can be shortened:

```python
            with self.subTest(params=params):
                with self.assertRaises(HTTPError):
                    get_versions(self.repo, params)
```